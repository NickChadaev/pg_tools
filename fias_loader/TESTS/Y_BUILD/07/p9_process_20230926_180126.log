2023-09-26 18:01:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_07 postgres /home/n.chadaev@abrr.local/Y_BUILD/07/stage_gar_c_07.csv 223-09-25
--------------------------------------------------------
[18:01:26] ... Кабардино-Балкарская Респ, id_region: 8, Execute: /home/n.chadaev@abrr.local/Y_BUILD/07/stage_gar_c_07.csv, process_id:5456
[18:01:26] ... Start process
[18:01:26] ...  -- Version
[18:01:26] ...  -- Очистка от данных
[18:01:26] ...  -- Set UNLOGGED
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_APARTMENT_TYPES_20230925_dec7f8ad-4238-4484-bf99-113a09c762f5.XML
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDR_OBJ_TYPES_20230925_4f714402-bdf0-4b03-8e2f-cda2704fb403.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ROOM_TYPES_20230925_6fedd268-7191-415b-be0e-8a5a110f7099.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OPERATION_TYPES_20230925_d65578ac-f625-4e50-ac1b-06ee9ac4de7a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_PARAM_TYPES_20230925_72393625-9278-4105-b37d-79dc852810ae.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_HOUSE_TYPES_20230925_1bdc09e7-b8f0-40d5-80f3-cf003785714b.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDHOUSE_TYPES_20230925_cf9caa45-fb3f-40d3-8ddf-770f536de4ed.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OBJECT_LEVELS_20230925_8ba71762-ad23-4052-b0b8-a659eac50bbb.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_TYPES_20230925_acf1cb3d-a66a-4e4b-b233-cc9c4032d98a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_KINDS_20230925_a8475f50-3b47-4725-9f46-b1bd99ea576c.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_REESTR_OBJECTS_20230925_3d09f1dc-16c1-4931-abfb-966915499144.XML
[18:05:01] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_ADDR_OBJ_20230925_5969f6f1-4682-44a1-aeb8-3218996026cb.XML
[18:05:05] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_HOUSES_20230925_b7647c73-29c8-4ff1-9ad7-f5aad3abec23.XML
[18:08:17] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_STEADS_20230925_4e9e9435-36d0-4565-b030-c738cff73d8a.XML
[18:08:23] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_APARTMENTS_20230925_389f4e40-c271-4fd6-b67b-cef57e57a545.XML
[18:09:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_ROOMS_20230925_8cf9bef9-9f45-4c13-8e9c-4ed00e25dc6b.XML
[18:09:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_CARPLACES_20230925_c52f8d45-cf59-400a-9f11-35b249ed597d.XML
[18:09:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_ADM_HIERARCHY_20230925_1bb5525f-ccb2-4584-822d-b21f012b3a7b.XML
[18:14:45] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_MUN_HIERARCHY_20230925_1b0a0efa-c6a9-4763-a418-70c7c035264e.XML
[18:18:56] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_CHANGE_HISTORY_20230925_91c9832a-437b-4063-8eac-4bc8d455b3b3.XML
[18:31:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_ADDR_OBJ_DIVISION_20230925_32bc9bad-c0fc-4432-99c9-689c095ead77.XML
[18:31:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_ADDR_OBJ_PARAMS_20230925_901f8181-b4d6-4e41-adcf-94bce06e18f2.XML
[18:32:09] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_STEADS_PARAMS_20230925_af1282ce-4dd6-4c43-be46-375f2907f946.XML
[18:33:20] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_HOUSES_PARAMS_20230925_583efb92-ad7f-4686-a4b4-8ead6f8dd39c.XML
[19:04:38] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_APARTMENTS_PARAMS_20230925_48834072-0047-4488-a4f2-2e7c6571dc0a.XML
[19:09:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_ROOMS_PARAMS_20230925_c816e907-73b2-4fde-b068-c25d9b4cb793.XML
[19:09:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_CARPLACES_PARAMS_20230925_fd2ef4ad-5804-4cd5-9440-67b47c9450fc.XML
[19:09:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/07/AS_NORMATIVE_DOCS_20230925_511c3cf0-3c96-4bc8-af57-c3b382935ebc.XML
[19:10:04] ...  -- Data finish
[19:10:04] ...  -- Set LOGGED
[19:10:16] ... Stop process
--------------------------------------------------------
