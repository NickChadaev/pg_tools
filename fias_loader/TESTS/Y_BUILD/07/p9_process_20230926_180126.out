 save_gar_version 
------------------
               70
(1 row)

CALL
CALL
 set_gar_dt_create 
-------------------
                70
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 306541
gar_fias.as_addr_obj: 5256
gar_fias.as_houses: 232875
gar_fias.as_steads: 8081
gar_fias.as_apartments: 110232
gar_fias.as_rooms: 87
gar_fias.as_carplaces: 0
gar_fias.as_adm_hierarchy: 383762
gar_fias.as_mun_hierarchy: 330784
gar_fias.as_change_history: 1072109
gar_fias.as_addr_obj_division: 12
gar_fias.as_addr_obj_params: 58334
gar_fias.as_steads_params: 99536
gar_fias.as_houses_params: 2506948
gar_fias.as_apartments_params: 425456
gar_fias.as_rooms_params: 97
gar_fias.as_carplaces_params: 0
gar_fias.as_normative_docs: 20106
